priya/javascript/Assignment6/JsAssign6Q3.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Array</title>
+    </head>
+    <body>
+        <h4>Enter 5 Numbers here</h4>
+        Number 1: <input type="text" id="num1" placeholder="Enter first Number "> <br>
+        <br>
+        Number 2: <input type="text" id="num2" placeholder="Enter second Number "> <br>
+        <br>
+        Number 3: <input type="text" id="num3" placeholder="Enter third Number "> <br>
+        <br>
+        Number 4: <input type="text" id="num4" placeholder="Enter forth Number "> <br>
+        <br>
+        Number 5: <input type="text" id="num5" placeholder="Enter fifth Number "> <br>
+        <br>
+        <button onclick="MyArray2()">Enter</button><br>
+        <br>
+        <button id="null">cancel</button>
+        <br>
+        <br>
+        <br>
+        <script language="javascript">
+           function MyArray2(){
+                var b,a,c,d,e,cancel,joined, arr;
```

Comment:
change variable names and there should be always a space after a comma so add it and make a change of variable